packages/toolset-revealer/valma/perspire.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env vlm
+
+// some web to node env emulation
+global.self = global;
+global.name = "Perspire window";
+global.window = global;
+
+const PerspireServer = require("@valos/inspire/PerspireServer").default;
```

Comment:
Nice, the separation is pretty much done.

Reply:
Change requests done. Only merge conflict remaining.

Reply:
just when I thought "oh this was painless transition." I noticed that I didn't babelified my changes so I didn't see any errors. Soo.. I remade every commit again. Anyways.. feature/perspire should be now rebased on top of prerelease-0.31.

Reply:
this change request reappeared during rebase: correcting it tomorrow.
import { PerspireView, createPerspireGateway } from "~/inspire";